### ISSUE TITLE
Disable restoring packages for single solution

### ISSUE THREAD

**Issue by tpluscode** [fsprojects/Paket.VisualStudio#104]
I'm using paket alpha Git support. Is it possible to disable the extension for one solution or make it use prerelease?

**Comment by forki** [comment]
Well that's an interesting issue. ;-)

Not sure how to handle it.

**Comment by Krzysztof-Cieslak** [comment]
@forki: Actually we have same problem in Ionide - for Paket integration we are always using latest release of Paket and there is no way to use prerelease.

**Comment by forki** [comment]
Yes. But I guess the real issue is that we use alpha features in real
projects ;-)

**Comment by Krzysztof-Cieslak** [comment]
Or someone creating really COOL feature and not releasing it in normal version ;P

**Comment by tpluscode** [comment]
@Krzysztof-Cieslak :+1: 
@forki and well, I'm using it because it works and I wanted to replace Gist with Git so that AppVeyor doesn't fail on GitHub rate limit

**Comment by forki** [comment]
I wonder if we should start building plugins against v3 branch ;-)

**Comment by Krzysztof-Cieslak** [comment]
It's one-liner change for Ionide :shrug:

**Comment by tpluscode** [comment]
Does the plugin use paket.bootstrapper.exe? Just allow the `prerelease` parameter

**Comment by forki** [comment]
Ionide yes. VS no.

In ionide one could add an easy switch. Here we need to move over to paket.exe instead of using paket.core.dll

**Comment by tpluscode** [comment]
Meh, maybe it's not worth the trouble. When is version 3 due?

**Comment by forki** [comment]
We need at least 2 months.
There is still much WIP

2016-03-03 14:33 GMT+01:00 Tomasz Pluskiewicz notifications@github.com:
